server/dash/dashboards/neighborhood.py

```diff
@@ -39,27 +42,80 @@ def populate_options():
         )
     return values
 
+print(populate_options())
```

Comment:
If we want to log this in prod, let's make it more detailed, e.g.,

`print("neighborhood.populate_options: output is ", populate_options())`

Reply:
thanks for catching this, this was for me to understand the code when I started

Reply:
I will get rid of this